src/components/Modal/component.stories.tsx

```diff
@@ -17,10 +19,34 @@ export const Default = () => {
         Show
       </Button>
       <Modal open={show} onClose={() => setShow(false)} data-testid="MyModal">
-        {new Array(200).fill(null).map((_, index) => (
-          <div key={index}>{index + 1}</div>
-        ))}
+        <Modal.Scroll>
+          <Modal.Body>{items}</Modal.Body>
+        </Modal.Scroll>
       </Modal>
     </>
   );
 };
+
+export const NoScroll = () => {
+  return (
+    <Modal open={true} onClose={() => {}}>
+      <Modal.Body>{items}</Modal.Body>
+    </Modal>
+  );
+};
+
+export const NoBody = () => {
+  return (
+    <Modal open={true} onClose={() => {}}>
+      <Modal.Scroll>{items}</Modal.Scroll>
+    </Modal>
+  );
+};
+
+export const Bare = () => {
+  return (
+    <Modal open={true} onClose={() => {}}>
```

Comment:
Why function is required? Just don't call it if it's undefined?

Reply:
It's not required, actually. I just left in the example for some reason.